File: practica1/estado.py
import transicion

class Estado:
    def __init__():
        self._Aceptacion=FALSE
        self._Transicion=set()
    
    def __init__(self,Aceptacion):
        self._Aceptacion=Aceptacion
        self._Transicion=set()

    def __init__(self,Aceptacion,Transicion):
        self._Aceptacion=Aceptacion
        self._Transicion=Transicion

    def __init__(self,Transicion):
        self._Aceptacion=FALSE
        self._Transicion=Transicion

File: practica1/afnPackage/Estado.java
import java.util.HashSet;

public class Estado{
  private boolean aceptacion;
  private HashSet<Transicion> transiciones;

  public Estado(){
    this.aceptacion=false;
    this.transiciones=new HashSet<>();
  }

  public Estado(boolean aceptacion){
    this.aceptacion=aceptacion;
    this.transiciones=new HashSet<>();
  }

  public Estado(HashSet<Transicion> transiciones){
    this.aceptacion=false;
    this.transiciones=transiciones;
  }

  public Estado(boolean aceptacion,HashSet<Transicion> transiciones){
    this.aceptacion=aceptacion;
    this.transiciones=transiciones;
  }

	public boolean isAceptacion() {
		return aceptacion;
	}

	public void setAceptacion(boolean aceptacion) {
		this.aceptacion = aceptacion;
	}

	public HashSet<Transicion> getTransiciones() {
		return transiciones;
	}

	public void setTransiciones(HashSet<Transicion> transiciones) {
		this.transiciones = transiciones;
	}
  
}
